### ISSUE TITLE
Size diff is wrong on Analytics PR

### ISSUE THREAD

**Issue by paulb777** [firebase/firebase-ios-sdk#8889]
![Screen Shot 2021-10-29 at 5 08 06 PM](https://user-images.githubusercontent.com/73870/139512986-878f994e-3831-4f40-999d-9ec5410c17a5.png)

**Comment by granluo** [comment]
Podfile:
```

source "https://github.com/firebase/SpecsStaging"
source "https://cdn.cocoapods.org/"

target 'SizeTest' do
 pod 'FirebaseAnalytics', :path => '/Users/runner/work/firebase-ios-sdk/firebase-ios-sdk'
end
```
```
Analyzing dependencies
Cloning spec repo `firebase-specsstaging` from `https://github.com/firebase/SpecsStaging`

Adding spec repo `trunk` with CDN `https://cdn.cocoapods.org/`
[!] Automatically assigning platform `iOS` with version `10.0` on target `SizeTest` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.
[!] CocoaPods could not find compatible versions for pod "GoogleAppMeasurement":
  In Podfile:
    FirebaseAnalytics (from `/Users/runner/work/firebase-ios-sdk/firebase-ios-sdk`) was resolved to 8.9.1, which depends on
      FirebaseAnalytics/AdIdSupport (= 8.9.1) was resolved to 8.9.1, which depends on
        GoogleAppMeasurement (= 8.9.1)

None of your spec sources contain a spec satisfying the dependency: `GoogleAppMeasurement (= 8.9.1)`.
```

That is probably caused by `GoogleAppMeasurement (= 8.9.1)` not uploaded to the [SpecsStaging repo](https://github.com/firebase/SpecsStaging) when the workflow was running.

The [workflow](https://github.com/firebase/firebase-ios-sdk/runs/4052308228?check_suite_focus=true) was running at 4:57 pm while `GoogleAppMeasurement` was uploaded at 6:15 pm on [the SpecsStaging repo](https://github.com/firebase/SpecsStaging/commit/51b50666577ee869bf473ac50a40717cca01800b)

**Comment by paulb777** [comment]
OK, that makes sense. Thanks for investigating.
